install/kgraph.sh

```diff
@@ -1,7 +1,11 @@
 git clone https://github.com/aaalgo/kgraph
-sudo apt-get install -y libboost-timer-dev libboost-chrono-dev libboost-program-options-dev libboost-system-dev libboost-python-dev
-cd kgraph
+pushd kgraph
+sudo make deps-ubuntu
 make
+<<<<<<< HEAD
```

Comment:
looks like a merge conflict here – can you resolve manually and resubmit? thanks!